packages/rocketchat-slackbridge/server/SlackAdapter.js

```diff
@@ -653,6 +656,29 @@ export default class SlackAdapter {
 		}
 	}
 
+	processFileShare(slackMessage) {
+		if (! RocketChat.settings.get('SlackBridge_FileUpload_Enabled')) {
+			return;
+		}
+
+		if (slackMessage.file && slackMessage.file.url_private_download !== undefined) {
+			const rocketChannel = this.rocket.getChannel(slackMessage);
+			const rocketUser = this.rocket.getUser(slackMessage.user);
+
+			//Hack to notify that a file was attempted to be uploaded
+			delete slackMessage.subtype;
+			slackMessage.text = `Uploaded a file: ${ slackMessage.file.permalink }`;
```

Comment:
Could we post this as an attachment to have the preview and remove that, not translated, text?